packages/react/src/components/UIShell/SideNavMenu.tsx

```diff
@@ -200,7 +200,9 @@ SideNavMenu.propTypes = {
 Defining the children parameter with the type ReactNode | ReactNode[]. This allows for various possibilities:
 a single element, an array of elements, or null or undefined.
 **/
-function hasActiveDescendant(children: ReactNode | ReactNode[]): boolean {
+export function hasActiveDescendant(
```

Comment:
If the tests are refactored to be [user centered](https://testing-library.com/docs/#:~:text=What%20you%20should,Child%20components) this won't need to be exported.